server/models/dtos/project_dto.py

```diff
@@ -89,6 +98,8 @@ class ProjectDTO(Model):
     priority_areas = BaseType(serialized_name='priorityAreas')
     last_updated = DateTimeType(serialized_name='lastUpdated')
     author = StringType()
+    task_creation_mode = StringType(required=True, serialized_name='taskCreationMode',
```

Comment:
Hi @pgiraud this is the source of the bug.  We are successfully creating the project using the ProjectAdminService.create_draft_project method, called from ProjectAdminAPI.put

However, we have a seperate API to allow the user to then modify their new project here ProjectAdminService.update_project(project_dto) called from the ProjectAdminAPI.post.  So the DTO is expecting that param to be there, but the front-end is not supplying it, meaning users can't update their project.

So I think we just need to remove the required=True, and the validators, as we want this property on the GET but we don't expect or need it on the POST.

Hope that makes sense.